userspace/engine/json_evt.h

```diff
@@ -59,41 +60,100 @@ class json_event : public gen_event
 	uint64_t m_event_ts;
 };
 
+// A class representing an extracted value or a value on the rhs of a
+// filter_check. This intentionally doesn't use the same types as
+// ppm_events_public.h to take advantage of actual classes instead of
+// lower-level pointers pointing to syscall events and to allow for
+// efficient set comparisons.
+
+class json_event_value
+{
+public:
+	enum param_type {
```

Comment:
Format according to the current C++ coding style pls.

Same need also in other places in the file

Reply:
I ran everything through clang-format.